src/components/skinManager.js

```diff
@@ -0,0 +1,184 @@
+define(['apphost', 'userSettings', 'browser', 'events', 'backdrop', 'globalize', 'require', 'appSettings'], function (appHost, userSettings, browser, events, backdrop, globalize, require, appSettings) {
```

Comment:
This file has been deleted in #1502

Reply:
I merged master into this branch and re-deleted the file (The previous rebase didn't remove it...)